banzai/tests/bias_utils.py

```diff
@@ -0,0 +1,20 @@
+from banzai.tests.utils import FakeContext, FakeImage
+import numpy as np
+from astropy.io import fits
+
+
+class FakeBiasImage(FakeImage):
+    def __init__(self, context=None, data=None, bias_level=0.0, nx=101, ny=103, header=None):
+        super(FakeBiasImage, self).__init__(image_multiplier=bias_level, nx=nx, ny=ny, pipeline_context=context)
+        if data is not None:
```

Comment:
I think these if-statements can be removed since they are also executed in `FakeImage`. And maybe the default header can be set before `super` is called, and then passed directly. This comment also applies to `FakeDarkImage` and `FakeFlatImage`.

Reply:
Done